.travis.yml

```diff
@@ -0,0 +1,25 @@
+env:
+  - NODE_VERSION=0.10
+  - NODE_VERSION=0.12
+
+os:
+  - linux
+  - osx
+
+install:
+  - git clone https://github.com/creationix/nvm.git /tmp/.nvm
+  - source /tmp/.nvm/nvm.sh
+  - nvm install $NODE_VERSION
+  - nvm use $NODE_VERSION
```

Comment:
Exactly (see https://github.com/travis-ci/travis-ci/issues/2311 for further information). To make it work consistently I ended up using a configuration similar to nodegit's one (https://github.com/nodegit/nodegit/blob/master/.travis.yml).